src/utils/KeyValues.ts

```diff
@@ -1,6 +1,7 @@
 export default class KeyValues {
   private static readonly digits = /[0-9]/
   private static readonly numbers = /[0-9.]/
+  private static readonly enterKeys = ['13', 'Enter']
```

Comment:
```suggestion
  public static readonly Enter = 'Enter'
```

Reply:
@alexnatalia cuz if we use `key` prop of keyboard event it will be `Enter`.

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key

Reply:
other props are deprecated